### PR TITLE
Fix: Different speeds don't work on Firefox mobile

### LINKED ISSUE #430: Different speeds don't work on Firefox mobile
When changing the speed to over 1, it plays faster for a little while, but every few seconds it loads even though the internet speed is fast enough to potentially play at >10x (150mbit). Doesn't happen in Chrome, but a fix would still be nice for the probably ~1000 (2%) people at TUM using Firefox Mobile.
Oddly enough happens more often in 1.25x and 1.5x than 2x. There are also no websites I've ever had this problem with so I assume TUM Live has to be the problem.
Tested in 99.2.0 and 101.0.0-beta.1

### PR DESCRIPTION
**Pull request by just-max**
Fixes #430 

Also removed (declared) dependency on `videojs-contrib-hls`, since HLS support is already built into VideoJS 7. As far as I can tell nothing is using the package, but correct me if I missed something.

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -42,7 +42,6 @@
     "moment": "^2.29.3",
     "nouislider": "^15.6.0",
     "video.js": "^7.18.1",
-    "videojs-contrib-hls": "^5.15.0",
```
Comment (joschahenningsen) [inline web/package.json:46]: can you also sync the package-lock.json?

File: web/template/admin/lecture-cut.gohtml [hunk 2]
```diff
@@ -20,7 +20,7 @@
     <div class="min-w-screen flex items-center justify-center">
         <div class="w-full lg:w-5/6">
             <p class="text-gray-300">Reference video:</p>
-            <video
+            <video-js
```
Comment (joschahenningsen) [inline web/template/admin/lecture-cut.gohtml:23]: I was thinking way to complicated here :smile: where did you find this fix? We should definitely do some testing as to how browsers react to this, but looks good!

**Review by joschahenningsen — CHANGES_REQUESTED** [review]
Amazing, thanks a lot for your engagement here, hit us up if you want to do this on a regular basis!

**Comment by just-max** [comment]
> where did you find this fix

the stream itself worked fine in the videojs demo, so from there transplanting things into the demo until it broke again :) the videojs [docs](https://videojs.com/guides/setup/#creating-a-player) do mention it, but of course I only saw that afterwards...

> hit us up if you want to do this on a regular basis

sure, depending on how busy the semester gets :<

**Review by joschahenningsen — APPROVED** [review]
Looks good! Thanks :)

**Pull request merged**